src/main/scala/cromwell/engine/ExecutionStatus.scala

```diff
@@ -2,11 +2,11 @@ package cromwell.engine
 
 object ExecutionStatus extends Enumeration {
   type ExecutionStatus = Value
-  val NotStarted, Starting, Running, Failed, Done, Aborted = Value
+  val NotStarted, Starting, Running, Failed, Preempted, Done, Aborted = Value
 
   implicit class EnhancedExecutionStatus(val status: ExecutionStatus) extends AnyVal {
     def isTerminal: Boolean = {
-      Seq(Failed, Done, Aborted) contains status
+      Seq(Failed, Done, Aborted, Preempted) contains status
```

Comment:
For this particular case I would change it to `Set` then because there is no ordering on the statuses